crypto/src/main/java/org/apache/tuweni/crypto/Hash.java

```diff
@@ -203,9 +202,9 @@ public static byte[] keccak256(byte[] input) {
    * @param input The input bytes to produce the digest for.
    * @return A digest.
    */
-  public static Bytes32 keccak256(Bytes input) {
+  public static Bytes keccak256(Bytes input) {
     try {
-      return (Bytes32) digestUsingAlgorithm(input, KECCAK_256);
+      return digestUsingAlgorithm(input, KECCAK_256);
```

Comment:
In this case, we're loosing the information that the returned Bytes from keccak256 is 32 bytes. I wonder if there're edge cases to this, as Bytes can store more than 32 bytes.